### PR TITLE
Add OpenSearch and available options for service forking

### PR DESCRIPTION
**Pull request by runwuf**
Add OpenSearch and available options for service forking
- OpenSearch was missing from the service forking documentation
- Added available service types for avn cli

### CODE REVIEW

File: docs/platform/howto/console-fork-service.rst [hunk 1]
```diff
@@ -24,10 +24,30 @@ Fork a service using the Aiven client (CLI)
...
+    kafka_connect
+    kafka_mirrormaker
+    m3db
+    m3coordinator
+    m3aggregator
+    mysql
+    opensearch
+    pg
+    redis
+
```
Comment (laysauchoa) [inline docs/platform/howto/console-fork-service.rst:46]: would be possible to show the command to list the services? I'm concerned that this list could become outdated soon if one of the services change.

Comment (runwuf) [reply docs/platform/howto/console-fork-service.rst:46]: It looks like the only way to show available options is to provide a type that doesn't exist...

> avn service create forked -t notype --plan someplan
> ERROR	command failed: UserError: Unknown service type 'notype' (available options: cassandra, clickhouse, elasticsearch, flink, grafana, influxdb, kafka, kafka_connect, kafka_mirrormaker, m3db, m3coordinator, m3aggregator, mysql, opensearch, pg, redis)

Comment (laysauchoa) [reply docs/platform/howto/console-fork-service.rst:46]: Thank you, I think it makes sense to link to the ``avn service types`` documentation then hardcode the services. What do you think @runwuf?

Comment (runwuf) [reply docs/platform/howto/console-fork-service.rst]: Thanks @laysauchoa and @lornajane! PR updated!

**Pull request merged**